content/_includes/page/list.njk

```diff
@@ -2,6 +2,8 @@
 {% import 'oss.macros.njk' as oss %}
 {% import 'events.macros.njk' as confs %}
 
+{%- set is_post = tags and ('_post' in tags) -%}
```

Comment:
This seems to be the same as `is_entry` defined above. I wonder if we should have these sort of things set in a more centralized/organized way. (I realize that's not really a new issue in this PR 😅)

Reply:
Oh yes, thanks for catching this! We can totally use `is_entry` now, in a previous version I was doing an additional check for 'Article' in the tags but now without that, we don't need this.